packages/plugins/src/hardhat/simulate.ts

```diff
@@ -210,7 +210,6 @@ export const simulateDeploymentSubtask = async (
   hre: any
 ): Promise<{
   receipts: Array<SphinxTransactionReceipt>
-  batches: Array<Array<SphinxLeafWithProof>>
```

Comment:
We don't use the `batches` for anything, so I removed it to simplify this function.